src/components/permissions/tests/testdata.ts

```diff
@@ -0,0 +1,312 @@
+const compileJWTdata = [
+  // Test JWT generation: 1
+  {
+    userInfo: {
+      userId: 1,
+      username: 'Check',
+    },
+    templatePermissionRows: [
+      {
+        permissiontType: 'Check',
+        templateCode: 'Check',
+        userId: 10,
+        username: 'Check',
+        permissionPolicyRules: {},
+        templatePermissionRestrictions: {
+          restrictOne: 1,
+          restrictTwo: 2,
+        },
+        templateId: 1,
+        permissionPolicyId: 2,
+        permissionNameId: 3,
+        templatePermissionId: 4,
+      },
+    ],
+    result: {
+      aud: 'postgraphile',
+      userId: 1,
+      username: 'Check',
+      pp2pn3: true,
+      pp2pn3tp4: true,
+      pp2pn3tp4_templateId: '1',
+      pp2pn3tp4_restrictOne: '1',
+      pp2pn3tp4_restrictTwo: '2',
+    },
+  },
+  // Test JWT generation: 2
+  {
+    userInfo: {
+      firstName: null,
+      lastName: null,
+      userId: 10,
+      username: 'userWithMultiplePermissions',
+      dateOfBirth: null,
+      email: null,
+    },
+    templatePermissionRows: [
+      {
+        permissionType: 'Apply',
+        permissionPolicyId: 1,
+        permissionPolicyRules: {},
+        permissionNameId: 1,
+        templatePermissionId: 1,
+        templatePermissionRestrictions: null,
+        templateId: 1,
+        templateCode: 'TestRego',
+        userId: 10,
+        username: 'userWithMultiplePermissions',
```

Comment:
I don't understand why the `username` is here as well

Reply:
It's an artifact, was previously there and was removed earlier in the PR and put back again for userInfo end point to work, Carl is changing that I think in his latest PR